packages/box/src/Stack.tsx

```diff
@@ -20,6 +20,9 @@ export const Stack = forwardRefWithAs<'div', StackProps>(function Stack(
 			flexDirection={flexDirection}
 			justifyContent={justifyContent}
 			alignItems={alignItems}
+			css={{
+				p: { margin: 0 },
+			}}
```

Comment:
Interesting, should we be expecting devs to drop any old element in a stack? Maybe the selector should  be for "any direct descendant of this component".

```suggestion
			css={{
				' > *': { margin: 0 },
			}}
```

Alternatively we don't handle this, and expect users to only put DS components inside a stack.